include/bluetooth/audio/mics.h

```diff
@@ -0,0 +1,367 @@
+ */
+int bt_mics_mute(struct bt_conn *conn);
+
+/**
+ * @brief Disable the mute functionality.
+ *
+ * Can be reenabled by called @ref bt_mics_mute or @ref bt_mics_unmute.
+ * This can only be done as the server.
+ *
+ * @return 0 on success, GATT error value on fail.
+ */
+int bt_mics_mute_disable(void);
+
+/**
+ * @brief Read the mute state of a Microphone Input Control Service.
+ *
+ * @param conn   Connection to peer device, or NULL to read local server value.
+ *
+ * @return 0 on success, GATT error value on fail.
+ */
+int bt_mics_mute_get(struct bt_conn *conn);
+
+/**
+ * @brief Read the Audio Input Control Service input state.
+ *
+ * @param conn          Connection to peer device,
+ *                      or NULL to read local server value.
+ * @param inst          Pointer to the Audio Input Control Service instance.
+ *
+ * @return 0 on success, GATT error value on fail.
+ */
+int bt_mics_aics_state_get(struct bt_conn *conn, struct bt_aics *inst);
+
+/**
+ * @brief Read the Audio Input Control Service gain settings.
+ *
+ * @param conn          Connection to peer device,
+ *                      or NULL to read local server value.
+ * @param inst          Pointer to the Audio Input Control Service instance.
+ *
+ * @return 0 on success, GATT error value on fail.
+ */
+int bt_mics_aics_gain_setting_get(struct bt_conn *conn, struct bt_aics *inst);
+
+/**
+ * @brief Read the Audio Input Control Service input type.
+ *
+ * @param conn          Connection to peer device,
+ *                      or NULL to read local server value.
+ * @param inst          Pointer to the Audio Input Control Service instance.
+ *
+ * @return 0 on success, GATT error value on fail.
+ */
+int bt_mics_aics_type_get(struct bt_conn *conn, struct bt_aics *inst);
+
+/**
+ * @brief Read the Audio Input Control Service input status.
+ *
+ * @param conn          Connection to peer device,
+ *                      or NULL to read local server value.
+ * @param inst          Pointer to the Audio Input Control Service instance.
+ *
+ * @return 0 on success, GATT error value on fail.
+ */
+int bt_mics_aics_status_get(struct bt_conn *conn, struct bt_aics *inst);
+
+/**
+ * @brief Unmute the Audio Input Control Service input.
+ *
+ * @param conn          Connection to peer device,
+ *                      or NULL to set local server value.
+ * @param inst          Pointer to the Audio Input Control Service instance.
+ *
+ * @return 0 on success, GATT error value on fail.
+ */
+int bt_mics_aics_unmute(struct bt_conn *conn, struct bt_aics *inst);
+
+/**
+ * @brief Mute the Audio Input Control Service input.
+ *
+ * @param conn          Connection to peer device,
+ *                      or NULL to set local server value.
+ * @param inst          Pointer to the Audio Input Control Service instance.
+ *
+ * @return 0 on success, GATT error value on fail.
+ */
+int bt_mics_aics_mute(struct bt_conn *conn, struct bt_aics *inst);
+
+/**
+ * @brief Set input gain to manual.
```

Comment:
This seems to be an exception in that it does not mention the AICS in the brief text